armi/reactor/converters/uniformMesh.py

```diff
@@ -102,25 +106,74 @@ def __init__(self, cs=None):
         # This prevents clearing out data on the original reactor that should
         # be preserved since no changes were applied.
         self._cachedReactorCoreParamData = {}
+        self._nonUniformMeshFlags = [
+            Flags.fromStringIgnoreErrors(f) for f in cs["nonUniformAssemFlags"]
+        ]
+        self._hasNonUniformAssems = any(self._nonUniformMeshFlags)
 
     def convert(self, r=None):
         """Create a new reactor core with a uniform mesh."""
         if r is None:
             raise ValueError(f"No reactor provided in {self}")
 
-        runLog.extra("Building copy of {} with a uniform axial mesh".format(r))
+        runLog.extra(f"Building with a uniform axial mesh reactor from {r}")
```

Comment:
This new runLog statement seems less clear to me. What was the motivation for changing it?

Reply:
In the case of using non uniform assemblies we don't copy the reactor.

Reply:
Got it. What do you think about having two `runLog` statements then? One for the `self._hasNonUniformAssems` case and one for the other? Something like... 
```python
if self._hasNonUniformAssems:
    runLog.extra(f"Temporarily replacing non-uniform assemblies in reactor, {r}, with assemblies whose axial mesh is uniform with r.core.refAssem.getAxialMesh().")
    ....
else:
    runLog.extra(f"Building copy of {r} with a uniform axial mesh.")
```